src/components/CallToAction.js

```diff
@@ -1,15 +1,20 @@
 import React from "react";
 import classNames from "classnames";
 import withStyles from "elevate-ui/withStyles";
-import CallToActionForm from "../components/CallToActionForm";
+import Link from "gatsby-link";
+import Button from "elevate-ui/Button";
 
 const CallToAction = ({ classes, className }) => (
   <div className={classNames(classes.root, className)}>
     <h1 className={classes.heading}>Take a tour</h1>
     <p className={classes.description}>
       Explore pure productivity with an Elevate Success Coach today
     </p>
-    <CallToActionForm className={classes.form} />
+    <Link className={classes.form} to="/get-started">
+      <Button innerClassName={classes.button} color="secondary">
```

Comment:
Can't use elevate-ui with Gatsby, something with `color.js` is breaking the build.

Also just a heads up- we won't ever want to nest a button inside of an `a` tag. In this case we can just style the Link to look like a button like we're doing elsewhere and we should be good to go. 👍